src/lib/state/multicall.tsx

```diff
@@ -33,20 +33,38 @@ function getBlocksPerFetchForChainId(chainId: number | undefined): number {
 export function MulticallUpdater() {
   const { chainId } = useWeb3React()
   const latestBlockNumber = useBlockNumber()
+  const latestEthBlockNumber = useEthBlockNumber()
   const contract = useInterfaceMulticall()
+  const ethContract = useEthOnlyInterfaceMulticall()
   const listenerOptions: ListenerOptions = useMemo(
     () => ({
       blocksPerFetch: getBlocksPerFetchForChainId(chainId),
     }),
     [chainId]
   )
+  const ethListener: ListenerOptions = useMemo(
+    () => ({
+      blocksPerFetch: getBlocksPerFetchForChainId(ChainId.MAINNET),
+    }),
+    []
+  )
 
   return (
-    <multicall.Updater
-      chainId={chainId}
-      latestBlockNumber={latestBlockNumber}
-      contract={contract}
-      listenerOptions={listenerOptions}
-    />
+    <>
+      <multicall.Updater
+        chainId={ChainId.MAINNET}
+        latestBlockNumber={latestEthBlockNumber}
+        contract={ethContract}
+        listenerOptions={ethListener}
+      />
+      {chainId !== ChainId.MAINNET && (
```

Comment:
should this render both multicalls? intuitively we're just on one chain at a time so i'd think its a ternary. 

also in general i think its good to do Multicall caps style for any react components

Reply:
yeah pretty much if we're on another chain (i.e. Polygon), we want a multicall for polygon to do the normal fetches as well as one for Mainnet to fetch ENS information.

Also wdym by caps style 😅

Reply:
`multicall` is lower-cased because it comes from `createMulticall` - the component is still upper-cased (`multicall.Updater`) - is that what you mean?

Reply:
Ah ok seems fine, I tend to like just keeping things uppercase when used as components but its a personal perference / nit, could de-structure it above but not worth the effort